### PR TITLE
feat: add repo organisation templates / jobs

### PR DESCRIPTION
**Pull request by ndom91**
### Changes

Adds a few things:
- PR Template
- Issue Templates
- PR Labeler config and GHA

The issue template will render similar to this:

image

A blank issue to start from is of course still allowed

### CODE REVIEW

**Comment by ndom91** [comment]
CC: @krlvi added some repo organisation type stuff here. Feel free to modify any of the copy or labels or whatever, but I usually find these little organisation -type things useful :)

File: .github/pr-labeler.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# https://github.com/actions/labeler#create-githublabeleryml
+rust: ["crates/**/*"]
+svelte: ["app/**/*"]
```
Comment (Caleb-T-Owens) [inline .github/pr-labeler.yml:3]: Very cool!

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+and link it via the following text:
+
+## 🎫 Affected issues
+
+Fixes: INSERT_ISSUE_NUMBER
+
+-->
+
+<!--
+If this is a WIP PR and you have todos left, feel free to uncomment this
```
Comment (Byron) [inline .github/PULL_REQUEST_TEMPLATE.md]:
```suggestion
If this is a WIP PR and you have todos left, feel free to uncomment this and turn this PR into a draft, see https://github.blog/2019-02-14-introducing-draft-pull-requests/
```

**Review by Byron — COMMENTED** [review]
That's lovely, it will be so useful. Particularly not having to ask for the version, distribution and platform anymore.

**Pull request merged**